docs/erp_statuscodes.adoc

```diff
@@ -119,6 +119,12 @@ h|Mögliche http Status Codes  h|Bedeutung/Fehlerdetails h|Verhalten im Fehlerfa
  |429	           |Zu viele Anfragen pro Zeiteinheit durch Nutzer | Client-Failover angeraten: Ja
 
  Ein Retry ist stündlich zulässig. (Max 10 Wiederholungen)
+ |454            |+ Die Anfrage wird abgebrochen, da die Voraussetzungen zur Erfüllung nicht gegeben sind. Der E-Rezept-Fachdienst akzeptiert zu diesem Zeitpunkt keine Anfragen mit einem Prüfungsnachweis, der das Ergebnis "3" enthält.| Client-Failover angeraten: Ja
```

Comment:
was wird unter Client-Failover verstanden?

Reply:
Wie ich das verstehe, ist das wenn der Client der andere FD Server versuchen sollten. Dann macht das auch kein Sinn in diesem Kontext. Ich verändere das auch